src/components/Onboarding.js

```diff
@@ -33,10 +32,7 @@ export const Onboarding = () => {
     { left: i18n.i18n('softkey-back'), onKeyLeft: prevOnboard, right: i18n.i18n('softkey-next'), onKeyRight: nextOnboard },
     { left: i18n.i18n('softkey-back'), onKeyLeft: prevOnboard, center: i18n.i18n('softkey-get-started'), onKeyCenter: exitOnboard }
   ]
-
-  useEffect(() => {
-    softkey.dispatch({ type: 'replace', config: softkeyConfig[currentIndex] })
-  }, [currentIndex])
+  useSoftkey('onboarding', { config: softkeyConfig[currentIndex], replace: true }, [currentIndex])
```

Comment:
Have you tried adding all the keys (for example: `center: ''` to blank the center key) instead of adding the "replace" option?

Reply:
use still able to trigger the function without seeing the center text.

it works if we have `{ center: '', onKeyCenter: () => {}}`, but then we will need to add a lot of empty key to clear the key ☹️

Reply:
This is how it is used in places where softkeys are conditional.
`center: currentLink ? i18n.i18n('centerkey-select') : '',`

I don't mind this change but it makes the configuration more complex for a case that is not too common. Could we maybe add a 4th argument that defaults to the old behavior? Something like `action='set'`

Reply:
for the conditional softkey config, we will have

```js
const softkeyConfig = {
    left: currentIndex > 0 ? i18n.i18n('softkey-back') : i18n.i18n('softkey-skip'),
    onKeyLeft: currentIndex > 0 ? prevOnboard : exitOnboard,
    center: currentIndex === 3 ? i18n.i18n('softkey-get-started') : '',
    onKeyCenter: currentIndex === 3 ? exitOnboard : () => {},
    right: currentIndex === 3 ? '' : i18n.i18n('softkey-next'),
    onKeyRight: currentIndex === 3 ? () => {} : nextOnboard
  }
```

it looks even more complex, I will go for the 4th argument, adding the action key